.github/workflows/node-ci.yml

```diff
@@ -80,21 +80,15 @@ jobs:
 
     defaults:
       run:
-        working-directory: ./
         shell: bash
 
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-
-      - name: Setup submodules
-        shell: bash
-        run: |
-          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
-          git submodule sync --recursive
-          git -c core.longpaths=true -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive || true
```

Comment:
just an fyi, there is a reason we did it this way. On the windows platform the normal checkout has issues with long file names, which this got around.

It may be possible that with the recent reorg of files that this path issue was no longer a problem, but we would have to look out for that.

Reply:
I remember this causing problems for me cloning on my windows system but I just did a recursive clone and it seems fine now. I checked and I still have long path support disabled at the moment.

If I recall correctly it was test fixtures in a submodule that had extremely long file names which caused problems.

Reply:
Thanks! I am indeed running into that problem when I do a recursive merge.